modules/generic_worker/templates/run-generic-worker.sh.multiuser.erb

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# This is a wrapper script for calling generic worker, such that when the worker
+# exits, this script will reboot the machine. Note, in the worker config
+# (/etc/generic-worker.config) we set numberOfTasksToRun to 1 so that the worker
+# will exit after running a single task. This script is then responsible for
+# rebooting the machine.
+
+# If this file exists when the worker starts up, and its content is equal
+# to numberOfTasksToRun, the worker won't exit and machine won't reboot.
+rm -f tasks-resolved-count.txt
```

Comment:
This `rm` and the comment above it is also not needed for multiuser when `numberOfTasksToRun` is `0`.